database/002_initial_schema.sql

```diff
@@ -388,13 +384,14 @@ GRANT CONNECT ON DATABASE booking_dev TO companyUsers,onlineUsers;
 GRANT SELECT (CID,email,username, name,orgNum) ON TABLE company TO companyUsers,onlineUsers;
 GRANT DELETE ON TABLE company TO onlineUsers;
 GRANT INSERT ON TABLE company TO onlineUsers;
-GRANT UPDATE (email,username,passhash) ON TABLE company TO companyUsers;
-GRANT USAGE, SELECT ON SEQUENCE company_cid_seq TO companyUsers;
+-- Do we really wanna give full admin to onlineUsers?
```

Comment:
As long as the onlineUsers are restricted to members of Bedkom I think it will be fine